### DISCUSSION TITLE
[A-L] (2023/24) Foglio 7 - Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#316] [Esercizi - Viaggi/Piazza]
![Screenshot 2023-11-12 150751](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/74c58def-460e-43f6-8f43-94f1656d8fe3)

**Comment by CasuFrost** [comment]
Se ho fatto qualche errore perfavore fatemelo notare.
![es algebra](https://github.com/sapienzastudentsnetwork/algebra2324/assets/97389762/15bcbd98-f42e-411c-ba1a-b07c423ca0e1)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/1fa206b5-2808-44da-9c0c-851c28764d87)
